File: test_scripts/r/test1/analysis.R
# Analysis script - depends on data_prep.R

source("data_prep.R")

# Perform analysis
summary_stats <- calculate_summary(clean_dataset)

# Group analysis
group_stats <- aggregate(value ~ category, clean_dataset, mean)

# Create a simple plot
if (require(graphics)) {
  png("analysis_plot.png")
  hist(clean_dataset$value, main = "Distribution of Values", xlab = "Value")
  dev.off()
}

print("Analysis completed")
print(summary_stats)

File: test_scripts/python/test1/analysis.py
"""Analysis script - depends on data_prep.py"""

import data_prep

# Perform additional analysis
dataset = data_prep.clean_dataset
stats = data_prep.stats

# Group analysis
above_average = [x for x in dataset if x > stats["mean"]]
below_average = [x for x in dataset if x <= stats["mean"]]

print(f"Analysis completed")
print(f"Above average: {len(above_average)} values")
print(f"Below average: {len(below_average)} values")
print(f"Mean: {stats['mean']:.2f}")